### PR TITLE
Add u-root chapter

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
yeah, it's still messy. Working on it.

File: 10.u-root/README.md [hunk 1]
```diff
@@ -0,0 +1,769 @@
...
+  this mode is surprisingly comfortable to use, given that program
+  compilation is so fast1.
+
+• Lockdown: if desired, the system can be locked down once booted in
+  one of several ways: the entire /src tree can be removed, for
+  example, or just the compiler toolchain can be deleted.
+
+### How u-root works
+
+U-root is packaged as an LZMA-compressed initial RAM file system
```
Comment (rjoleary) [inline 10.u-root/README.md]: I hear xz is the recommended now.

File: 10.u-root/README.md [hunk 2]
```diff
@@ -0,0 +1,769 @@
...
+environments. In response to the increasing security challenges
+facing embedded systems in the always-connected Internet of Things, we
+have chosen to write all the u-root programs in Go, a modern, type
+safe language with garbage collection. The safety of the Go language
+and runtime reduce many of the security risks of writing
+network-facing services. The performance of the Go compiler makes
+on-demand compilation practical: most commands compile in a fraction
+of a second and, once compiled, run in about a millisecond.
+
+The U-root file system, on boot, contains only 5 binaries. The rest
```
Comment (rjoleary) [inline 10.u-root/README.md]: It's less than 5 binaries now.

File: 10.u-root/README.md [hunk 3]
```diff
@@ -0,0 +1,769 @@
+# u-root, what it is and why it is
```
Comment (rjoleary) [inline 10.u-root/README.md]: Some work is needed to bring this up to date and formatted correctly. I feel like we can submit this now and work through the issues in a separate PRs.

**Pull request merged**